qt/python/mantidqt/CMakeLists.txt

```diff
@@ -62,24 +62,24 @@ if(ENABLE_WORKBENCH)
 
   mtd_add_sip_module(MODULE_NAME _commonqt5
                      TARGET_NAME mantidqt_commonqt5
-                     SIP_SRCS _common.sip
                      HEADER_DEPS
-                       ${_header_depends}
```

Comment:
Sorry to step in here. I was just looking over the changes to check how something was used and I spotted this. I think `${_header_depends}` needs to be moved back to after the `HEADER_DEPS` keyword? If not I think this will break the target updating when a header changes.

Reply:
That is a great spot! I must have missed it.